src/api/src/create_book.py

```diff
@@ -133,7 +133,7 @@ def generate_clean_part_html(part: Part, content: str) -> bs4.Tag:
 
     clean = BeautifulSoup(
         f"""
-    <section id="section_{chapter_id}" class="chapitre">
+    <section id="section_{chapter_id}" class="chapter">
```

Comment:
You can get rid of class, as neither chaptire nor chapter are defined in our stylesheet.